hypervisor/arch/x86/Kconfig

```diff
@@ -343,6 +343,13 @@ config ENFORCE_TURNOFF_AC
 	  If CPU has #AC for split-locked access, HV enable it and VMs can't disable.
 	  Set this to enforce turn off that #AC, for community developer only.
 
+config ENFORCE_TURNOFF_GP
+	bool "Force to disable #GP for UC lock"
+	default n
+	help
+	  If CPU has #GP for UC lock, HV enable it and VMs can't disable.
+	  Set this to enforce turn off that #GP, for community developer only.
+
 config IVSHMEM_ENABLED
```

Comment:
@taoyuhong  Kconfig is no longer being used. These codes have no effect...

Please apply the Kconfig change editing following files:
misc/config_tools/schema/config.xsd
misc/config_tools/schema/config_common.xsl
and add a node <ENFORCE_TURNOFF_GP>n</ENFORCE_TURNOFF_GP>  under <hv>/<FEATURES> to all applicable $SCENARIO.xmls.

Reply:
Have discard Kconfig change